terraform/main.tf

```diff
@@ -0,0 +1,58 @@
+data "azurerm_resource_group" "rg" {
+  name = var.resource_group_name
+}
+
+module "networking" {
+  source              = "./modules/networking"
+  vnet_name           = var.vnet_name
+  resource_group_name = data.azurerm_resource_group.rg.name
+  location            = var.location
+  address_space       = var.address_space
+  subnets = [
+    {
+      name   = var.subnet_name
```

Comment:
Nu am apucat sa discutam despre asta dar aici am avea doua optiune pe care sa le folosim cand dam valori la variabile:
1. Punem toate valorile aici in main.tf (ex: name = "exm-subnet-1")
2. Lasam cum e acum si creem un fisier .tfvars unde dam valori la variabile (despre asta va trebui sa va uitati putin cum se face)
Prima varianta e mai usoara dar cu a doua varianta mai invatati ceva
@en-nader-said